pkcs7_test.go

```diff
@@ -317,7 +317,7 @@ func UnmarshalTestFixture(testPEMBlock string) TestFixture {
 			result.Input = derBlock.Bytes
 		case "CERTIFICATE":
 			result.Certificate, _ = x509.ParseCertificate(derBlock.Bytes)
-		case "PRIVATE KEY":
+		case "PRIVATE KEY", "RSA PRIVATE KEY":
```

Comment:
The consensus is to use `PRIVATE KEY` for PKCS#8 and `RSA PRIVATE KEY` for PKCS#1.

Reply:
Adding `"RSA PRIVATE KEY"` was required for the test cases from the existing PRs that I ported over to here.

Reply:
Your change is ok, but if you get PEM with `PRIVATE KEY` is very likely that is encoded using PKCS#8

```
$ openssl genrsa -out rsa.pem
$ head -1 rsa.pem
-----BEGIN RSA PRIVATE KEY-----
$ openssl pkcs8 -topk8 -in rsa.pem -nocrypt | head -1
-----BEGIN PRIVATE KEY-----
```

Reply:
I'm ok leaving this for backward compatibility. I don't think we are using this path.

Reply:
It's also just in a `_test.go` file to extract that from the hardcoded PEMs. We could replace the tests if we want to.